clang-tools-extra/clang-tidy/cppcoreguidelines/MissingStdForwardCheck.cpp

```diff
@@ -129,15 +131,25 @@ void MissingStdForwardCheck::registerMatchers(MatchFinder *Finder) {
       unless(anyOf(hasAncestor(typeLoc()),
                    hasAncestor(expr(hasUnevaluatedContext())))));
 
+  auto StaticCast =
+      Options.get("IgnoreStaticCasts", false)
```

Comment:
thats not a valid way to add options, please move this to constructor, and change storeOptions.

Reply:
Ah right, thanks.